src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class Car implements Comparable<Car> {
+    private static final int cutLine = 4;
```

Comment:
상수의 네이밍은 `CamelCase`보다는 상수 네이밍 규약을 따르는 게 어떨까요? 
```suggestion
    private static final int CUT_LINE = 4;
```